### ISSUE TITLE
suggestion: custom group and mode of sshd_config

### ISSUE THREAD

**Issue by patemery67** [saz/puppet-ssh#274]
I have a need to set the group of the sshd_config to something other than root (aka gid 0) and change the mode to allow group members to read the file.

Specifically, we are looking to run a product from Venafi that will manage user ssh keys. Some element of the product requires the ability to read the sshd_config. This drives the need for the above customization and capability.

**Comment by patemery67** [comment]
I was thinking that if you agree with this idea I'd like to take a stab at implementing these mods with some collaboration and/or guidance on your part. What do you think?

**Comment by saz** [comment]
Go ahead, I'm open for any PR but it should be backwards compatible

**Comment by patemery67** [comment]
Saz,
I've got some code changes in a fork in my userspace ([my fork](https://github.com/patemery67/puppet-ssh/tree/fix_issue_274)) that I'd like to have you review. I don't know what the usual etiquette is for this. Do I submit a pull request and wait for comments or do I somehow send you a link to the branch in my work space or what? I know that I need to add documentation ot the README but I don't want to do that until I'm sure you agree to the naming and placement of the parameters I have added.
Thanks,
Pat
